arho_feature_template/core/forms/feature_attribute_form.py

```diff
@@ -0,0 +1,13 @@
+from qgis.PyQt.QtWidgets import QDialog
+
+from arho_feature_template.core.feature_template import FeatureTemplate
+from arho_feature_template.qgis_plugin_tools.tools.resources import load_ui
+
+
+class FeatureAttributeForm(QDialog, load_ui("feature_attribute_form.ui")):
```

Comment:
I don't have strong preference for the UI loading, but I like the idea that the ui files are next to the python module. Changed to this (but gave `__package__` as an argument for` files()` since it seems to be needed in older Python versions).